db/migrate/20200605183957_add_start_time_access_to_section_due_dates.rb

```diff
@@ -0,0 +1,5 @@
+class AddStartTimeAccessToSectionDueDates < ActiveRecord::Migration[6.0]
+  def change
+    add_column :section_due_dates, :start_time, :datetime, null: true
```

Comment:
Should this also have a `default: nil`?

Reply:
I don't see why not. The current behaviour is for datetimes with `null: true` to default to `nil` but I can't find any rails documentation to back that up (though this is the [correct behaviour for postgres](https://www.postgresql.org/docs/10/ddl-default.html)) so maybe better safe than sorry if that gets changed from under us.